lib/big_brother/node.rb

```diff
@@ -30,7 +30,7 @@ def incorporate_state(another_node)
     end
 
     def invalidate_weight!
-      @weight = nil
+      @weight = 0
```

Comment:
There are multiple assumptions in the current code that weight is a numeric object and it seems that 0 is the full down / invalid value elsewhere.  Trying to make the least amount of change using 0 here to indicate an unused backend would be optimal.

The IPVS wlc and rr schedulers in Linux kernel 3.18.52 treat weight 0 nodes as down if we're mocking that.
* https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/net/netfilter/ipvs/ip_vs_rr.c?h=v3.18.52#n58
* https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/net/netfilter/ipvs/ip_vs_wlc.c?h=v3.18.52#n34
* https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/net/netfilter/ipvs/ip_vs_wrr.c?h=v3.18.52

```
cluster.rb:          node.weight.zero?
cluster.rb:      total_health = @nodes.collect{ |n| n.weight || 0 }.reduce(:+)
cluster.rb:      nodes_down = @nodes.count{|n| n.weight == 0}
cluster.rb:        node.weight = 0
node.rb:      return 1 if self.weight.to_i.zero?
node.rb:      return -1 if other.weight.to_i.zero?
node.rb:      if !@max_weight.nil? && @max_weight.is_a?(Integer) && @max_weight > 0 && @max_weight < health
```